### PR TITLE
Document dashboard configuration

### PR DESCRIPTION
**Pull request by rudigiesler**
It would be helpful to document how to configure a dashboard to use metrics fired via the metrics helper.

### CODE REVIEW

**Comment by rudigiesler** [comment]
@hodgestar ready for review

File: docs/dashboard.rst [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+method `avg`, which stores the average time in milliseconds taken to get from
+the `enter` event of the `states:start` state to the `enter` event of the
+`states:coffee` state.
+
+The following is an example of using this metric within the Vumi Go Dashboard:
+
+.. code-block:: javascript
+
+    {
+        "type": "diamondash.widgets.lvalue.LValueWidget",
```
Comment (hodgestar) [inline docs/dashboard.rst]: Do you think it would be better to show the time between things as a GraphWidget rather?

**Comment by hodgestar** [comment]
One small question, otherwise looks good.

**Comment by rudigiesler** [comment]
@hodgestar Ready for review again.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**